.github/workflows/functional-test.yml

```diff
@@ -0,0 +1,28 @@
+name: Functional Test
+
+on:
+  push:
+    paths-ignore:
+      - 'docs/**'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+    - run: git config --global user.email "ask-cli-test@amazon.com"
+    - run: git config --global user.name "Ask Cli Test"
```

Comment:
is this for hosted skill?

Reply:
yes. cannot git push until set up name and email